<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="ssidLabel" >
     <property name="text" >
      <string>SSID</string>
     </property>
     <property name="buddy" >
      <cstring>ssidEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3" >
    <widget class="QLineEdit" name="ssidEdit" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="encLabel" >
     <property name="text" >
      <string>Encryption</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3" >
    <widget class="QCheckBox" name="ssidHexCheck" >
     <property name="text" >
      <string>hexadecimal chars</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QComboBox" name="encCombo" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text" >
       <string>CCMP</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>TKIP</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Channel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <widget class="QComboBox" name="channelCombo" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text" >
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>13</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>14</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>PSK</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3" >
    <widget class="QLineEdit" name="pskEdit" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
